repository-data/application/src/main/resources/hcm-config/namespaces/hippostd/html.yaml

```diff
@@ -2,9 +2,9 @@
 definitions:
   config:
     /hippo:namespaces/hippostd/html/editor:templates/_default_:
-      ckeditor.config.appended.json: '{ toolbarGroups: [{ name: ''document''}], extraPlugins:
-        ''balloonpanel,a11ychecker,abbr,definedterms,wordcount'', allowedContent:
-        ''h2;h3;h4;code;em;strong;ul;div;span;thead;tbody;td;tr;th;caption;ol;li;dfn;abbr;sub;sup;img[*](*);a[*](*);p{text-align};abbr[title,id];span[class];table[class,id,dir,data-disablesort]{*};iframe[id,class,style,title,height,name,src,width,allowfullscreen,sandbox,srcdoc]{*}'',
+      ckeditor.config.appended.json: '{ toolbarGroups: [{ name: ''document'', items:
+        [ ''PickImage'']]}, extraPlugins: ''balloonpanel,a11ychecker,abbr,definedterms,wordcount'',
+        allowedContent: ''h2;h3;h4;code;em;strong;ul;div;span;thead;tbody;td;tr;th;caption;ol;li;dfn;abbr;sub;sup;img[*](*);a[*](*);p{text-align};abbr[title,id];span[class];table[class,id,dir,data-disablesort]{*};iframe[id,class,style,title,height,name,src,width,allowfullscreen,sandbox,srcdoc]{*}'',
```

Comment:
Looks like incorrect JSON

After much discussion we find the json is broken somewhere else so we must break it first :trollface: